setup.py

```diff
@@ -84,8 +84,8 @@ def __call__(self):
     # List run-time dependencies here. These will be installed by pip when your project is
     # installed. For an analysis of "install_requires" vs pip's requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
-    install_requires=["uproot>=3.11.1", "matplotlib>=3.1.2", "pandas>=0.25.3", "scikit-learn>=0.22.1",
-                      "xgboost>=0.90,<1.0", "shap>=0.34", "bayesian-optimization>=1.0.1"],
+    install_requires=["uproot>=3.11.1", "matplotlib>=3.2", "pandas>=1.0", "scikit-learn>=0.23",
+                      "xgboost>=1.1.1,<1.2", "shap>=0.35", "bayesian-optimization>=1.2"],
```

Comment:
Let's bump to xgboost 1.0.x and all the other packages while we keep an eye of future shap releases. In this way the new release will anyway allow to have newer packages in the dependency chain